snafu/stressng_wrapper/trigger_stressng.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import subprocess
+import time
+import logging
+import yaml
+
+logger = logging.getLogger("snafu")
+
+class Trigger_stressng():
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.runtype = args.runtype
+        self.timeout = args.timeout
+        self.vm_stressors = args.vm_stressors
+        self.vm_bytes = args.vm_bytes
+        self.mem_stressors = args.mem_stressors
+
+    def _run_stressng(self):
+        cmd = "stress-ng --job /workload/jobfile --log-file stressng.lg -Y stressng.yml"
```

Comment:
I think we should parametrize the --job file path, it would be specially useful to run stress-ng outside a container